Extras/Serialize/BulletFileLoader/btBulletFile.cpp

```diff
@@ -13,6 +13,7 @@ subject to the following restrictions:
 3. This notice may not be removed or altered from any source distribution.
 */
 
+#include "Bullet3Serialize/Bullet2FileLoader/bullet2FileLoaderApi.h"
```

Comment:
This dependency on that header is not wanted, please remove it.

Reply:
This is removed in my most recent force push.